### ISSUE TITLE
 check for errors - validation of number of subtags

### ISSUE THREAD

**Issue by Bogiegit** [fisharebest/webtrees#4621]
Tags should be checked in terms of minumum/maximum number of subtags

some samples:

- Tag 'CAUS' may only appear 1 times under tag 'DEAT'
- Tag 'DATE' may only appear 1 times under tag 'BIRT', 'DEAT', 'MARR'
- Tag 'NICK' may only appear 1 times under tag 'NAME'
- Tag 'PLAC' may only appear 1 times under tag 'OCCU'
- Tag 'TYPE' may only appear 1 times under tag 'EVEN', 'MARR', 'NAME'

**Comment by reteP-riS** [comment]
This is a double-edged sword... Sometimes you would want to document contradictions by using the same subtag twice with the first one being the preferred one.
Nevertheless it would be nice if webtrees showed a **warning**, not necessarily an error.

**Comment by fisharebest** [comment]
I do not think that is permitted.  You can enter multiple facts/events - but not lower-level attributes.

For example, if you had two contradictory `INDI:DEAT:CAUS`, then you would need two `INDI:DEAT` events

**Comment by reteP-riS** [comment]
Good point - thanks!
